webapp/pom.xml

```diff
@@ -0,0 +1,1532 @@
+            <version>${gs.version}</version>
+          </dependency>
+          <dependency>
+            <groupId>org.geoserver.community</groupId>
+            <artifactId>gs-wps-longitudinal-profile</artifactId>
+            <version>${gs.version}</version>
+          </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>mbstyle</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.geoserver.extension</groupId>
+          <artifactId>gs-mbstyle</artifactId>
+          <version>${gs.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>ogcapi</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.geoserver.extension</groupId>
+          <artifactId>gs-ogcapi-features</artifactId>
+          <version>${gs.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.geoserver.extension</groupId>
+          <artifactId>gs-web-features</artifactId>
+          <version>${gs.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.geoserver.extension</groupId>
+          <artifactId>gs-web-ogcapi</artifactId>
+          <version>${gs.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>web-resource</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.geoserver.extension</groupId>
+          <artifactId>gs-web-resource</artifactId>
+          <version>${gs.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>flatgeobuf</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.geoserver.community</groupId>
+          <artifactId>gs-flatgeobuf</artifactId>
+          <version>${gs.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>debianPackage</id>
+      <properties>
+        <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>pl.project13.maven</groupId>
+            <artifactId>git-commit-id-plugin</artifactId>
+  	        <version>4.0.0</version>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>revision</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <prefix>build</prefix>
+              <failOnNoGitDirectory>false</failOnNoGitDirectory>
+              <skipPoms>false</skipPoms>
+              <verbose>false</verbose>
+              <gitDescribe>
+                <tags>true</tags>
+              </gitDescribe>
+              <injectIntoSysProperties>true</injectIntoSysProperties>
+            </configuration>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
```

Comment:
thinking out loud, but im not even sure it makes sense to ship https://github.com/georchestra/datadir/tree/master/geoserver which is useless and hasnt been updated in 10 years. nothing in the geoserver code reads files from `/etc/georchestra`... wdyt @pmauduit ?

Reply:
I think it's still being used by geofence-enabled versions.

Reply:
also I like the `geoserver.properties` file explaining that `the princess is in another castle`, as it documents a bit why there is quite nothing in the georchestra datadir.